js/src/modal.js

```diff
@@ -83,6 +92,7 @@ class Modal {
     this._ignoreBackdropClick = false
     this._isTransitioning     = false
     this._scrollbarWidth      = 0
+    this._touchSupported      = 'ontouchstart' in document.documentElement || navigator.maxTouchPoints > 0
```

Comment:
@Johann-S: maybe we should have a common function in util about this?

Reply:
@XhmikosR & @Johann-S 
I can add a util method if you like...
```
    this._touchSupported      = Util.isTouchSupported()
```

or would a "constant" (like `Util.TRANSITION_END` be preferred ?
```
    this._touchSupported      = Util.TOUCH_SUPPORTED
```

if going with the "constant" route, would we even need the local `this._touchSupported` assignment and instead simply check `Util.TOUCH_SUPPORTED` vs `this._touchSupported`?

Reply:
A constant should be enough 👍

Reply:
done..  dropdown.js & tooltip.js also do a touch test... now also using Util.isTouchEnabled()
I was getting unit test errors when going the full constant route..  I don't know why...  but seemed to be carousel unit test related